packages/piro/test/Main_AnalysisDriver_Tempus.cpp

```diff
@@ -239,10 +239,10 @@ int main(int argc, char *argv[]) {
         }
         else if (mockModel=="MassSpringDamperModel") {
           auto comm = appComm->createSubcommunicator(std::vector<int>({0}));
-          RCP<Thyra::ModelEvaluator<double>> model_tmp = rcp(new MassSpringDamperModel(comm,false,probParams,true));
+          RCP<Thyra::ModelEvaluator<double>> model_tmp = rcp(new MassSpringDamperModel(comm,false,probParams,true,probParams->get<bool>("Use x_dot in g", false)));
```

Comment:
This option is used to select the objective function used inside the mass spring damper model.
It could be used to choose between `x_dot[0]` and `x[1]` as the value of the speed of the mass.
Both of them are valid cases for both integrated responses over time and responses at the final time step.